### ISSUE TITLE
support convert from darknet-yolov2 to caffe?

### ISSUE THREAD

**Issue by ysh329** [marvis/pytorch-caffe-darknet-convert#28]
Thanks in advance! 🤣

**Comment by nvnnghia** [comment]
how did you convert darknet yolov2 ti caffe? i am interesting :D :D

**Comment by ysh329** [comment]
@nvnnghia check this: https://github.com/ysh329/deep-learning-model-convertor/issues/24 and using my code here: https://github.com/ysh329/pytorch-caffe-darknet-convert

**Comment by nvnnghia** [comment]
I tried your code but got this error
`I0119 13:48:32.472975  9657 net.cpp:406] layer27-act <- layer27-conv
I0119 13:48:32.472980  9657 net.cpp:367] layer27-act -> layer27-conv (in-place)
I0119 13:48:32.472985  9657 net.cpp:122] Setting up layer27-act
I0119 13:48:32.473003  9657 net.cpp:129] Top shape: 1 64 26 26 (43264)
I0119 13:48:32.473014  9657 net.cpp:137] Memory required for data: 271005696
I0119 13:48:32.473024  9657 layer_factory.hpp:77] Creating layer layer28-reshape
I0119 13:48:32.473033  9657 net.cpp:84] Creating Layer layer28-reshape
I0119 13:48:32.473039  9657 net.cpp:406] layer28-reshape <- layer27-conv
I0119 13:48:32.473047  9657 net.cpp:380] layer28-reshape -> layer28-reshape
F0119 13:48:32.473073  9657 reshape_layer.cpp:87] Check failed: top[0]->count() == bottom[0]->count() (165888 vs. 43264) output count must match input count
*** Check failure stack trace: ***
Aborted (core dumped)
`

Any suggestion? I tried to convert yolo-voc model.
